docs/pages/machine-id/guides/ansible.mdx

```diff
@@ -55,21 +54,23 @@ remote_tmp=/tmp
 
 [ssh_connection]
 scp_if_ssh = True
-ssh_args = -F ./ssh.cfg
+ssh_args = -F /opt/machine-id/ssh_config
```

Comment:
I think it might help clarify/emphasize the usefulness of Machine ID to call out the fact that we're using a Machine ID-managed directory in the sentence "Create a file `ansible.cfg`". Maybe something like:

```markdown
Create a file called `ansible.cfg`. We will configure Ansible 
to run the OpenSSH client with the configuration file generated 
by Machine ID, `/opt/machine-id/ssh_config`.
```